www/main/overlay/components/CopyToClipboard.jsx

```diff
@@ -6,22 +6,42 @@ const copy = require('copy-to-clipboard');
 
 const { i18n } = remote.require('./app');
 
-export const CopyToClipboard = ({ url }) => (
-  <div className="overlay-url">
-    <p className="overlay-window-text live-url-header">{i18n.t(`BANI_OVERLAY.LIVE_URL`)}</p>
-    <div className="url-container">
-      <input disabled={true} type="text" className="url-text" value={url} />
-      <span
-        className="export-btn"
-        onClick={() => {
-          copy(url);
-        }}
-      >
-        <i className="fa fa-files-o cp-icon"></i>
-      </span>
+export const CopyToClipboard = ({ url }) => {
+  const [copied, setCopied] = React.useState(false);
+
+  const handleCopy = () => {
+    copy(url);
+    setCopied(!copied);
+    setTimeout(() => setCopied(false), 2000); // Reset the copied state after 2 seconds.
```

Comment:
Instead of doing `setTimeout` what if we use the mouseleave event?
https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event

So the flow would be, 
- by default the value of tooltip is **Copy URL**.
- After click it changed to **Copied!**
- On mouseleave event from that copy icon, change tooltip value back to Copy URL.

Currently, it shows Copied, and if I keep my mouse cursor there, it changes back to Copy URL.

![overlay-copy](https://github.com/KhalisFoundation/sttm-desktop/assets/1990932/66d54d5c-22ac-43ae-8d7f-617bec43bf9c)